python/packages/ai/teams/memory_fork.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
can we separate this into its own file called `memory.py`?

Reply:
Missed [this file](https://github.com/microsoft/teams-ai/blob/PYTHON/python/packages/ai/teams/state/memory.py) but turns out `Memory` has already been implemented so updated the import to this instead